### ISSUE TITLE
Drawing in a pictureBox on secondary Lua forms doesn't work

### ISSUE THREAD

**Issue by Deakula** [TASEmulators/BizHawk#3778]
### Summary
When a pictureBox is created with the handle of a form that is **NOT** the _oldest_ form created in the Lua script, and then an attempt is made to draw on said pictureBox, the drawing is unsuccessful. The "oldest" form being the form that was created before all others with the exception of calling `forms.destroy()` on the oldest form causing the second-oldest form to take its place and so on.

### Repro
1. Create two or more forms with calls to `forms.newform()`
2. Create a pictureBox with `forms.pictureBox()` for any/all of said forms (at least one of which being a form that is **NOT** the oldest form created).
3. Attempt to call any draw method (such as `forms.drawLine()` or `forms.drawText()`) for any/all of the pictureBoxes.

### Output
For each attempt to draw on a pictureBox that does **NOT** belong to the oldest form, the following is outputted to the Lua console:
```
Drawing functions can only be used on PictureBox components.
```

### Host env.
- BizHawk 2.9.1; Win11 Home 22H2; Intel/NVIDIA

### Screenshots

![Second Form PictureBox](https://github.com/TASEmulators/BizHawk/assets/21248445/bd78c02d-e4fc-43b6-849f-ba531d5e8b53)

![Second Form PictureBox 2](https://github.com/TASEmulators/BizHawk/assets/21248445/8c6186d5-6ec4-44a8-9add-6503c62908c8)

![Second Form PictureBox 3](https://github.com/TASEmulators/BizHawk/assets/21248445/6ee32922-0a3d-4192-ab8b-bf868da4ce36)

**Comment by YoshiRulz** [comment]
I'm guessing no-one (including me) ever tested the linked fix—should be good with https://github.com/TASEmulators/BizHawk/commit/681a5764d6da02b374c439e42df86a76c7916be0 (though of course I haven't tested that either).

**Comment by Deakula** [comment]
> I'm guessing no-one (including me) ever tested the linked fix—should be good with [681a576](https://github.com/TASEmulators/BizHawk/commit/681a5764d6da02b374c439e42df86a76c7916be0) (though of course I haven't tested that either).

(Late, but) I've tested all the same code in 2.10 release as in my original issue, and I can confirm that everything is now working as expected. Thank you.
